src/mongoc/mongoc-cursor-private.h

```diff
@@ -46,7 +46,13 @@ struct _mongoc_cursor_interface_t
    void             (*get_host) (mongoc_cursor_t        *cursor,
                                  mongoc_host_list_t     *host);
 };
+#ifdef MIN
+#undef MIN
+#endif
 
+#ifdef MAX
+#undef MAX
+#endif
```

Comment:
I feel this is a bit unfortunate. we should probably rename the macros below instead